autopush_rs/src/client.rs

```diff
@@ -435,9 +435,10 @@ where
         Ok(Async::Ready(item))
     }
 
-    fn process_hello(&mut self, uaid: Uuid, message_month: String, reset_uaid: bool, rotate_message_table: bool) -> ClientState {
+    fn process_hello(&mut self, uaid: Uuid, message_month: String, check_storage: bool, reset_uaid: bool, rotate_message_table: bool) -> ClientState {
```

Comment:
nit: Again, I'd be happier if the new arg was at the end of this list.

Reply:
Ah yes, here it matters! Changing.